annotator/src/test/java/org/cbioportal/annotator/mixin/VariantAnnotationMixin.java

```diff
@@ -0,0 +1,35 @@
+package org.cbioportal.annotator.mixin;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.genome_nexus.client.TranscriptConsequence;
+import org.genome_nexus.client.VariantAnnotationSummary;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class VariantAnnotationMixin
```

Comment:
This mixin is only needed within the test scope. All other mappings are handled by the auto generated API client.

Reply:
same here